R/learn.R

```diff
@@ -119,9 +122,12 @@ mssql_learn_query <- function() { # taken directly from {datamodelr}
       and ind_col.column_id = cols.column_id
     left outer join sys.systypes [types] on
       types.xusertype = cols.system_type_id
+  where tabs.schema_id = schema_id('%s')
```

Comment:
```suggestion
  where tabs.schema_id = schema_id(%s)
```

Reply:
I tried this and your next suggestion in all 3 combinations, but it won't work, not even with normal schema names. 
Probably the reason is that here we do not want to address the schema in a direct way, but we want to refer to the schema name representation in the result of the query. If we use `'%s'`, it is even possible to address schemas like "SELECT" or other keywords.

> Otherwise the SQL is bad and prone to injection if schema contains '.

Since I am not able to create a schema containing `'`, I unfortunately can't test this corner case.

Reply:
This is what `dbQuoteIdentifier()` is supposed to do.

Reply:
I mean `dbQuoteString()` -- you're right that `dbQuoteIdentifier()` is incorrect here.